org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java

```diff
@@ -212,17 +217,41 @@ public void triggerInitialization(Collection<IPath> roots) {
 				logException(e.getMessage(), e);
 			}
 		}
+		// load maven plugin https://github.com/redhat-developer/vscode-java/issues/2088
+		try {
+			long start = System.currentTimeMillis();
+			JavaLanguageServerPlugin.logInfo("Starting " + IMavenConstants.PLUGIN_ID);
+			Platform.getBundle(IMavenConstants.PLUGIN_ID).start(Bundle.START_TRANSIENT);
+			JavaLanguageServerPlugin.logInfo("Started " + IMavenConstants.PLUGIN_ID + " " + (System.currentTimeMillis() - start) + "ms");
+		} catch (BundleException e) {
+			logException(e.getMessage(), e);
+		}
```

Comment:
this can be extracted to a common method such as `void startBundle(String bundleName)`

Reply:
Fixed.